controllers/repo_manager/README.md

```diff
@@ -14,6 +14,7 @@
 * [PulpSpec](#pulpspec)
 * [PulpStatus](#pulpstatus)
 * [Redis](#redis)
+* [ResourceManager](#resourcemanager)
```

Comment:
we don't need it anymore

Reply:
hum... I had to add the ResourceManager field into the CRD to keep compatibility with ansible version (we still have it in the old CRD).
But I added a `[DEPRECATED]` "label" to let users aware of it:
https://github.com/pulp/pulp-operator/blob/8e674110c6c200850f01edaf75c1b6925b79cc2f/controllers/repo_manager/README.md?plain=1#L281-L289

Reply:
I believe we don't have users in the version that requires it anymore, wdyt @rooftopcellist ?